pkg/controllers/vdb/install_reconcile.go

```diff
@@ -329,7 +329,7 @@ func (d *InstallReconciler) genCmdRemoveOldConfig() []string {
 }
 
 // acceptEulaInPod will run a script that will accept the eula in the given pod
-func (d *InstallReconciler) acceptEulaInPod(ctx context.Context, pf *PodFact) error {
```

Comment:
I suggest moving this to a new file eula.pod, as its not install specific anymore.